src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000042> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01000277>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000042> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_03000041>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000043> (material transformation process)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000043> "A process during which the one or more materials, present within a given site, transform into another one or more materials.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000043> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000043> "material transformation process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000043> <http://purl.obolibrary.org/obo/ENVO_02500000>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000043> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002229> <http://purl.obolibrary.org/obo/ENVO_03000010>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000044> (seawater freezing process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://www.waterencyclopedia.com/Re-St/Sea-Water-Freezing-of.html") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000044> "A material transformation process during which seawater freezes into sea ice.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000044> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000044> "seawater freezing process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> <http://purl.obolibrary.org/obo/ENVO_03000043>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00002149>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00002200>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000045> (brine rejection process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Brine_rejection") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000045> "A material transformation process during which brine is formed due to the expulsion of salt during frazil formation or a seawater freezing process. Expelled salt accumulates in brine droplets, which remain in a liquid state due to their increased salinity.")
```

Comment:
@pbuttigieg broke second sentence of definition into a comment.
Def = 
> A material transformation process during which brine is formed due to the expulsion of salt during frazil formation or a seawater freezing process. 

comment:
> Expelled salt accumulates in brine droplets, which remain in a liquid state due to their increased salinity.